install/install_openadapt.ps1

```diff
@@ -55,8 +59,7 @@ function RunAndCheck {
 function Cleanup {
     $exists = Test-Path -Path $setupdir
     if ($exists) {
-        Set-Location $setupdir
-        Set-Location ../
+        Set-Location $env:USERPROFILE
         Remove-Item -LiteralPath $setupdir -Force -Recurse
```

Comment:
I see you used $openAdaptPath as a new setup dir where we will run the OpenAdapt's installation.

Should not this line be changed to:
`Remove-Item -LiteralPath $openAdaptPath -Force -Recurse` ?


Also on L298 and L299 we have a set-location to $setupdir. Should we change that to to $openAdaptPath as well ?

Reply:
My intention behind the changes is to use separate folders for separate tasks.
My proposal is for:
- the download and installation of dependencies to happen from `$setupdir`;
- the git clone and installation of OpenAdapt to happen from `$openAdaptPath`.

So in theory, someone who has the correct versions of all 3 dependencies (Tesseract, Python and Git) already installed would only have the install script clone and setup OpenAdapt from git.

Someone who has 2 out of the 3 dependencies would have the missing dep. installed by the script.

Someone who has the unsupported version of Python would have the correct version installed etc.